controllers/articles.js

```diff
@@ -0,0 +1,8 @@
+const {selectArticleById} = require('../models/articles');
+
+exports.getArticleById = (req, res, next) => {
+    const {article_id} = req.params;
+    return selectArticleById(article_id)
+    .then(article => res.status(200).send({article}))
+    .catch(err => next(err));
```

Comment:
because all you're doing is this catch is invoking next, you could take advantage of an express shorthand and write this as `.catch(next)` which will achieve the same thing